client/src/modules/Profile/ProfilePage.js

```diff
@@ -1,5 +1,96 @@
 import React from 'react';
+import Card from 'antd/lib/card';
+import Badge from 'antd/lib/badge';
+import Icon from 'antd/lib/icon';
+import { Label } from 'office-ui-fabric-react/lib/Label';
+const centerContent = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center'
+};
 
-export default () => {
-	return <div>Perfil</div>
+const profileImg = {
+	zIndex: '9',
+	'width': '15%',
+	'position': 'absolute',
+	top: '50%',
+    left: '5%',
+    borderRadius: '90px'
+};
+
+
+export default class Profile extends React.PureComponent{
+
+	eventsCount(title,count){
```

Comment:
eventsCount () {} With spaces.

Also the naming of the function is not correctly spelled.